tests/plugins/trim_whitespace.test.ts

```diff
@@ -0,0 +1,73 @@
+import { Model } from "../../src";
+import { getCellContent } from "../test_helpers";
+import { selectCell, setCellContent, setSelection } from "../test_helpers/commands_helpers";
+import { createModelFromGrid, getRangeValuesAsMatrix } from "../test_helpers/helpers";
+
+describe("trim whitespace", () => {
+  test("trim cell content", () => {
+    const model = new Model();
+    setCellContent(model, "A2", "   Alo         ");
+    selectCell(model, "A2");
+    model.dispatch("TRIM_WHITESPACE");
+    expect(getCellContent(model, "A2")).toBe("Alo");
+  });
+
+  test("remove duplicate spaces", () => {
+    const model = new Model();
+    setCellContent(model, "A2", "  Alo        salut     sunt  eu    un haiduc  ");
+    selectCell(model, "A2");
+    model.dispatch("TRIM_WHITESPACE");
+    expect(getCellContent(model, "A2")).toBe("Alo salut sunt eu un haiduc");
+  });
+
+  test("keep lines break", () => {
```

Comment:
test with an empty line. (it should be removed, but it's not)

Reply:
It was chosen to keep empty lines.
A test was added for this